fili-core/src/test/groovy/com/yahoo/bard/webservice/table/availability/AvailabilityTestingUtils.groovy

```diff
@@ -62,4 +64,21 @@ class AvailabilityTestingUtils extends Specification {
                     table.setAvailability(new ConcreteAvailability(table.getTableName(), table.getSchema().getColumns(), new TestDataSourceMetadataService(allIntervals)));
                 }
     }
+
+    /**
+     * Construct a map of Column object to its all available intervals
+     *
+     * @param map a map of Column in String representation to its all available intervals in String representations
+     *
+     * @return a map of Column object to its all available intervals
+     */
+    static def constructColumnToIntervalMap(Map<String, Set<String>> map) {
+        return map.entrySet().stream()
+                .collect(
+                    Collectors.toMap(
+                            {it -> new Column(it.key)},
+                            {(it.value).each {new Interval(it)}}
+                    )
```

Comment:
This entire method body can be simplified using Groovy's collection extensions directly:

```groovy
map.collectEntries{key, value -> [ (new Column(key)): value.collect{new Interval(it)} ]}
```